packages/clay-date-picker/src/index.tsx

```diff
@@ -445,10 +580,11 @@ const ClayDatePicker: React.FunctionComponent<IProps> = React.forwardRef<
 										{({day, key}) => (
 											<DayNumber
```

Comment:
Regarding this behavior change in `DayNumber` it is still valid considering my previous comment about it https://github.com/liferay/clay/pull/4012#discussion_r612004289. We can talk more tomorrow because you didn't follow through.